app/assets/stylesheets/components/navigation.scss

```diff
@@ -112,4 +112,26 @@
       }
     }
   }
+
+  &--item {
+    padding: $su-3;
+    text-align: center;
+    width: 100%;
+
+    &.selected {
+      border-bottom: 4px solid var(--accent-brand);
+
+      .crayons-tabs--label {
+        color: var(--card-color);
```

Comment:
I'd use one of the `--link-*` variables instead of `--card-color` since card is, in most cases, specifically made for `.crayons-card`.

Reply:
Resolved in latest commits.